lib/chef/provisioning/aws_driver/aws_provider.rb

```diff
@@ -276,7 +276,7 @@ def wait_for_status(aws_object, expected_status, acceptable_errors = [], tries=6
     expected_status = [expected_status].flatten
     current_status = aws_object.status
 
-    Retryable.retryable(:tries => tries, :sleep => sleep, :on => StatusTimeoutError) do |retries, exception|
```

Comment:
This change doesn't include a backoff.  It only changes it so we catch 'id not found' errors and retry those too